### DISCUSSION TITLE
Deleting a Google Analytics property

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#18] [Q&A] [answered]
Guidelines on deleting a Google Analytics property

**Comment by andatoshiki** [comment, marked as answer]
Delete a Google Analytics property
Important: Your property, and all the reporting views in the property, will be permanently deleted 35 days after being moved to the Trash Can. Once deleted, the property is gone, and we are not able to retrieve any historical data or reinstate your reports.

You must have the [Editor role ](https://support.google.com/analytics/answer/2884495)to move a property to the Trash Can. Moving a property to the Trash Can also moves all component views to the Trash Can. To move a property to the Trash Can:

[Sign in to Google Analytics.](https://analytics.google.com/).
Click [Admin](https://support.google.com/analytics/answer/6132368), and navigate to the [property you want to delete](https://support.google.com/analytics/answer/6099198).
In the PROPERTY column, click Property Settings.
Click Move to Trash Can.
On the confirmation screen, click Move property to Trash Can.
